addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1902,17 +1902,49 @@ var SnippetsMenu = Widget.extend({
             const $invisibleSnippets = globalSelector.all().find('.o_snippet_invisible').addBack('.o_snippet_invisible');
 
             $invisibleDOMPanelEl.toggleClass('d-none', !$invisibleSnippets.length);
-
-            const proms = _.map($invisibleSnippets, async el => {
-                const editor = await this._createSnippetEditor($(el));
+            // childrenPerSnippet -key: a snippet.
+            //                    -value: an array filled with its children
+            // snippets.
+            const childrenPerSnippet = new Map();
+            // Filter the $invisibleSnippets to only keep the non children
+            // snippets and create the map (childrenPerSnippet) of the snippets
+            // and their children.
```

Comment:
Not sure to understand what is a `non children snippet` :thinking:

Reply:
I renamed it as `root snippet` :+1: